src/org/jitsi/impl/neomedia/transform/AbsSendTimeEngine.java

```diff
@@ -129,8 +130,52 @@ private void setTimestamp(byte[] buf, int off)
      * disable this transformer.
      * @param id the ID to set.
      */
-    public void setExtensionID(int id)
+    public void setExtensionID(byte id)
     {
         extensionID = id;
     }
+
+    /**
+     *
+     * @param packet is the RawPacket
+     * @returns the Absolute Send Time header extension if present
+     */
+    public HeaderExtension getAbsoluteSendTimeExtension(RawPacket packet)
+    {
+        if( packet.getExtensionBit() && packet.getHeaderExtensionType()
```

Comment:
We should check whether the AbsSendTimeEngine is configured with an extension ID (e.g. getExtensionID() != -1).